content/guides/use-case/wiremock.md

```diff
@@ -0,0 +1,272 @@
+
+ 3. Start the Compose stack by running the following command at the root of the cloned project directory
+
+    ```console
+    $ docker compose up -d
+    ```
+
+    After a moment, the application will be up and running. 
+
+    ![Diagram showing the WireMock container running on Docker Desktop ](./images/wiremock-using-docker.webp)
+
+
+    You can check the logs by selecting the **WireMock** container:
+
+    ![Diagram showing the logs of WireMock container running on Docker Desktop ](./images/wiremock-logs-docker-desktop.webp)
+
+ 4. Test the Mock API.
+
+    ```console
+    $ curl http://localhost:8080/api/v1/getWeather\?city\=Bengaluru
+    ```
+ 
+    It will return the following response:
+
+    ```plaintext
+    {"city":"Bengaluru","temperature":27.1,"conditions":"Mostly cloudy","forecasts":[{"date":"2024-09-02T07:00:00+05:30","temperature":83,"conditions":"Partly sunny w/ t-storms"},{"date":"2024-09-03T07:00:00+05:30","temperature":83,"conditions":"Thunderstorms"},{"date":"2024-09-04T07:00:00+05:30","temperature":83,"conditions":"Intermittent clouds"},{"date":"2024-09-05T07:00:00+05:30","temperature":82,"conditions":"Dreary"},{"date":"2024-09-06T07:00:00+05:30","temperature":82,"conditions":"Dreary"}]}
+    ```
+
+    You get this response due to the mapping file in the `wiremock-endpoint/mappings/getWeather/getWeatherBengaluru.json` file. 
+
+
+## Using WireMock in development
+
+   Now that you have tried WireMock, let’s use it in development and testing. In this example, you will use a sample application that has a Node.js backend. This app stack has the following configuration:
+
+  * **Local Development Environment**: The context in which the Node.js backend and WireMock are running.
+  * **Node.js Backend**: Represents the backend application that handles HTTP requests.
+  * **External AccuWeather API**: The real API from which live weather data is fetched.
+  * **WireMock**: The mock server that simulates the API responses during testing. It runs as a Docker container.
+
+  ![Diagram showing the architecture of WireMock in development ](./images/wiremock-arch.webp)
+
+  * The API Request is sent from the Node.js backend to the External AccuWeather API, which then returns the API Response.
+  * Alternatively, when using WireMock, a Mocked API Request is sent to the WireMock server, and a Mocked API Response is returned.
+
+  In this guide, you’ll learn how to:
+
+  * Set up a non-containerized Node app to route requests to the containerized WireMock 
+  * Set up a non-containerized Node app to fetch data from the external AccuWeather API.
+
+  The application can be accessed at [dockersamples/wiremock-node-docker](https://github.com/dockersamples/wiremock-node-docker).
+
+
+## Set up a non-containerized Node app to route requests to a containerized WireMock
```

Comment:
Is this meant to be a third-level section?

Reply:
Removed the statement as it is not required.